docs/examples.rst

```diff
@@ -853,7 +853,46 @@ exactly is happening during test execution. This also allows to
 inspect particular place of the code by inserting a ``bash`` in
 the shell code or ``import pdb; pdb.set_trace()`` for python::
 
-    tmt run --all execute --how beakerlib.tmt --interactive
+    tmt run --all execute --how tmt --interactive
+
+
+Aliases
+------------------------------------------------------------------
+
+It might be useful to setup a set of shell aliases for the tmt
+command lines which you often use. For a quick reservation of
+a machine or a container for quick experimenting::
+
+    alias reserve='tmt run login --step execute execute finish provision --how container --image fedora'
+
+Reserving a testing box then can be as short as this::
+
+    reserve
+    reserve -h virtual
+    reserve -i fedora:32
+    reserve --how virtual
+    reserve --image fedora:32
+
+For interactive debugging of tests the following three aliases can
+come handy::
```

Comment:
I think this should rather be `come in handy`, I personally haven't seen `come handy` and haven't found anything in a dictionary about it

Reply:
Fixed.